components/FlexibleForms/TextField.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Field as RawField,
+  ErrorMessage,
+  getIn,
+  FormikErrors,
+  FormikTouched,
+  FormikValues,
+} from 'formik';
+
+interface FieldProps {
+  touched: FormikTouched<FormikValues>;
+  errors: FormikErrors<FormikValues>;
+  name: string;
+  label: string;
+  type?: string;
+  hint?: string;
+  className?: string;
+  required?: boolean;
```

Comment:
You have `required` in the type here but don't seem to use it?

Reply:
holdover from an older attempt at validation